packages/ts-sdk-apm/src/index.ts

```diff
@@ -1,2 +1,4 @@
 export * from '../types/types.d'
 export { default as createTelemetryMiddleware } from './apm'
+export { default as createTelemetryMiddlewareV2 } from './apm-legacy'
```

Comment:
v2 usually means newer to me. It's conflicting with the legacy, which means older.